gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/zosmf/ZosmfService.java

```diff
@@ -557,12 +557,13 @@ public JWKSet getPublicKeys() {
         final String url = getURI(getZosmfServiceId(), authConfigurationProperties.getZosmf().getJwtEndpoint());
 
         try {
-            final String json = restTemplateWithoutKeystore.getForObject(url, String.class);
-            return JWKSet.parse(json);
+            return JWKSet.load(new URL(url));
         } catch (ParseException pe) {
             log.debug("Invalid format of public keys from z/OSMF", pe);
```

Comment:
should these messages be `debug` or rather `error`?

Reply:
I would leave them in the debug, this exception would not have serious implication on the functionality